scripts/docgen/theme/partials/member.sources.hbs

```diff
@@ -0,0 +1,11 @@
+<aside class="tsd-sources">
```

Comment:
Does it work because it overwrites the default behavior and doesn't include link to source?

Reply:
Yeah, it is identical to the Typedoc template file it's overriding except the source link section is removed.